server/game/cards/05-LoCR/TheRainsOfCastamere.js

```diff
@@ -50,10 +51,12 @@ class TheRainsOfCastamere extends AgendaCard {
         this.game.addMessage('{0} uses {1} and kneels their faction card to reveal {2}',
             context.player, this, context.target);
 
+        this.game.startRevealingPlots();
```

Comment:
The changes to the plot phase were necessary (in my opinion) because currently reactions to a plot card entering the used pile during the plot phase triggers BEFORE the resolution of the When Revealed effects of the chosen plots. This is wrong from a rule perspective. You can only trigger Sand Steed/Tattered Prince´s Reaction AFTER all When Revealed effects have been resolved.
I tried to figure out how to make the whole "revealing of new plots and placing the old one in the used pile" atomic, but this didn´t solve the issue or i wasn´t trying the correct changes, so i went for a distinction between placing plots in the used pile during the revelation of plots in the plot phase and not.
I´ve tested the changes with Tatteres/Sand Steed and Rains and also with Change of Plans (which moves plots to the used pile without the revealing part) and so far i haven´t encountered any bugs/sideeffects

Reply:
Reverted these changes in favor of #3073, which has actual simultaneous reaction windows.